models/urn.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "fmt"
+
+func TableUrn(service, host, database, name string) string {
+	return fmt.Sprintf("%s::%s/%s/%s", service, host, database, name)
+}
+
+func DashboardUrn(service, host, id string) string {
```

Comment:
For `Urn`, I think better to make it capitalized `URN` like the standard case in golang. ``

[Ref](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

```suggestion
func DashboardURN(service, host, id string) string {
```